$OpenBSD: patch-Lite_Util_Makefile_PL,v 1.1 2015/08/17 18:29:15 bluhm Exp $

https://rt.cpan.org/Public/Bug/Display.html?id=106489

test_cc(), as changed in NetAddr-IP-4.077/Lite/Util/Makefile.PL,
does no longer find C compiler.  within the foreach loop, test
should be for $CC, not $_

--- Lite/Util/Makefile.PL.orig	Fri Aug 14 09:32:04 2015
+++ Lite/Util/Makefile.PL	Mon Aug 17 19:55:35 2015
@@ -214,7 +214,7 @@ sub test_cc {
 
   my $CC;
   foreach $CC (($ENV{CC},$Config{ccname},$Config{cc})) {
-    next unless $_;
+    next unless $CC;
     my $command = qq|$CC compile.c -o compile.output|;
 
     print F <<'EOF';
